admin/templates/preprint_providers/detail.html

```diff
@@ -36,11 +39,7 @@
                 {% if perms.osf.change_preprintprovider %}
                     <a class="btn btn-primary" href={% url 'preprint_providers:register_moderator_admin' preprint_provider.id %}>Register Moderator/Admin</a>
                     <a class="btn btn-primary" href={% url 'preprint_providers:edit' preprint_provider.id %}>Modify Preprint Provider</a>
-                {% if show_taxonomies %}
                     <a class="btn btn-primary" href={% url 'preprint_providers:process_custom_taxonomy' preprint_provider.id %}>Modify Custom Taxonomy</a>
-                {% else %}
-                    <a class="btn btn-primary disabled">Modify Custom Taxonomy</a>
```

Comment:
Wouldn't this make sense to be "View Custom Taxonomy" when you're not allowed to edit it?